### ISSUE TITLE
Change the offset of the mutation

### ISSUE THREAD

**Issue by AlokPatel99** [AlokPatel99/Flappy-Bird-using-Neural-Network-and-Genetic-Algorithm#12]
I changed the mutation offset from (-0.1, 0.1) to (-0.25, 0.25). It is working well for me, the bird is getting a train. You can experiment with other values.  Still, in my opinion, the crossover will help the agent to learn fast.

**Comment by rbehal** [comment]
Yeah actually, I did not realize that random.randn is a Gaussian distribution I thought it was random numbers between 0 and 1. It might be better to actually just change it from a Gaussian distribution from 0-1 and keep the lower offset. The reason is because the inputs are between 0 and 1 as well.
